### PR TITLE
Add CHIPs page

### PR DESCRIPTION
**Pull request by merc1er**

### CODE REVIEW

File: app/templates/base.njk [hunk 1]
```diff
@@ -54,16 +54,14 @@
...
       <div class="container text-center">
-        <a class="text-white" href="/$( _lang_ )$/bitcoin-cash-website-notice">
-          $( website-notice.banner.title )$<br>
-          $( website-notice.banner.sub )$
+        <a class="text-white" href="/$( _lang_ )$/">
         </a>
       </div>
     </div>
-  </div>
+</div> -->
```
Comment (emergent-reasons) [inline app/templates/base.njk:64]: Leaving this here in case we need a banner in the future?

File: app/templates/footer.njk [hunk 2]
```diff
@@ -49,7 +52,7 @@
             <a href="/$( _lang_ )$/media" class="nav-link text-muted">$( footer.media )$</a>
           </li>
           <li class="nav-item">
-            <a href="/$( _lang_ )$/tools/" class="nav-link text-muted">$( footer.tools-tools )$ <span class="badge badge-primary">New</span></a>
```
Comment (emergent-reasons) [inline app/templates/footer.njk:52]: What's this removing?

File: app/templates/base.njk (hunk 1 above)
Comment (merc1er) [reply app/templates/base.njk:64]: Yes, that was the idea.

File: js/tags.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        $(this).addClass('d-none');
+      }
+    });
+  }
+  if (showError) {
+    $('#tag-error').removeClass('d-none');
+  } else {
+    $('#tag-error').addClass('d-none');
+  }
+});
```
Comment (emergent-reasons) [inline js/tags.js:45]: Wow nice. I didn't know you had gone this far.

File: app/templates/footer.njk (hunk 2 above)
Comment (merc1er) [reply app/templates/footer.njk:52]: The "new" badge, because the address and broadcast tools aren't really new anymore

**Review by emergent-reasons — APPROVED** [review]
Just had a few questions but LGTM

**Pull request merged**